src/templates/default_function/config.toml

```diff
@@ -1,8 +1,8 @@
-# The values in this file will be used if flags aren't given
-# in CLI commands. Can create multiple --environments (default 'dev')
+# Can create multiple --environments (default 'dev')
```

Comment:
Frankly speaking I'm not sure if this file is useful in any way.

Reply:
I think it is once we explain why we create this file in docs 😄